hasher-matcher-actioner/hmalib/common/mappings.py

```diff
@@ -24,11 +24,35 @@
 from hmalib.indexers.s3_indexers import (
     S3BackedMD5Index,
     S3BackedPDQIndex,
+    S3BackedPDQFlatIndex,
     S3BackedInstrumentedIndexMixin,
 )
 
 # Maps from signal type → index to use for that signal type.
-INDEX_MAPPING: t.Dict[t.Type[SignalType], t.Type[S3BackedInstrumentedIndexMixin]] = {
-    PdqSignal: S3BackedPDQIndex,
-    VideoMD5Signal: S3BackedMD5Index,
+INDEX_MAPPING: t.Dict[
+    t.Type[SignalType], t.List[t.Type[S3BackedInstrumentedIndexMixin]]
+] = {
+    PdqSignal: [S3BackedPDQIndex, S3BackedPDQFlatIndex],
+    VideoMD5Signal: [S3BackedMD5Index],
 }
+
+
+def get_index_for_signal_type_matching(
```

Comment:
Any arguments for or against pushing this to the `get_index()` method on the `Matcher` class in matchers_base.py?